src/geometry/manifold/algebra/left_invariant_derivation.lean

```diff
@@ -0,0 +1,175 @@
+/-
+Copyright © 2020 Nicolò Cavalleri. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Nicolò Cavalleri
+-/
+
+import geometry.manifold.derivation_bundle
+import ring_theory.derivation
+
+/-!
+
+# Left invariant derivations
+
+In this file we define the concept of left invariant derivation for a Lie group. The concept is the
+analogous to the more classical concept of left invariant vector fields, and this analogy "commutes"
+with the isomorphism between global derivations and left invariant vector fields.
```

Comment:
I don't understand what "commutes" could mean here. Could you please spell this out?